extensions/amp-story-player/0.1/test/validator-amp-story-player.html

```diff
@@ -29,6 +29,7 @@
 <body>
   <amp-story-player width="360" height="600">
     <a href="https://www.example.com" class="story">
+      <img data-amp-story-player-poster-img src="https://www.example.com/img" width="360" height="600" loading="lazy">
```

Comment:
could we mark this as a valid usage and could we include an invalid usage in the test file?

Reply:
I added another file `validator-amp-story-player-error` with an invalid usage and `validator-amp-story-player-img` with valid usage. Is that ok?

Reply:
That's great, thank you.